pages/index.tsx

```diff
@@ -158,6 +164,9 @@ export const getStaticProps: GetStaticProps = async context => {
   await queryClient.prefetchQuery(['subpages', [locale, 'announcement']], () =>
     getSubpages({ locale, type: 'announcement' }),
   )
+  await queryClient.prefetchQuery([locale, 'blog'], () =>
+    getSubpages({ locale, type: 'blog' }),
+  )
```

Comment:
```suggestion
  await queryClient.prefetchQuery(['posts', [locale]], () =>
    getBlogPosts(locale),
  )
```

Make sure to give the same query key, so that it will always check the same key for cached data.

See below how we set queryKey for blogPosts
https://github.com/Samenvvv/frontend/blob/d58358682d129fc189ddfd05438d466d7c221752/src/lib/getData/getBlogPosts.ts#L43